### PR TITLE
finished week-3 assignment of NodeJS

### PR DESCRIPTION
**Pull request by choudhary-keshav**
Pull Request Questionnaire

(1) Which Jira ticket(s) is this pull request for, and is the ticket now complete?

(2) Briefly describe your change.
I have pushed my week-3 assignment in this nodeJS project. In my week-2 assignment I have used firebase CRUD operations & multer for saving files on server and used JEST for testing.

(3) What parts of the application are affected by your change?
Nothing

(4) How did you test your change? If relevant, which role(s) and/or customer configuration(s) did you use?
Tested my application on various edge cases and some common scenarios

### CODE REVIEW

File: src/Week3/firebaseUser.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      email: req.body.email,
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+    };
+    const userSnapshot = await db.collection("users").doc(user.email).get();
+    if (userSnapshot.exists) {
+      return res.status(404).json({ error: "User already exists, use some other email" });
+    }
+    const response = await db.collection("users").doc(id).set(user);
+    res.send(response);
```
Comment (aditya-s-19) [inline src/Week3/firebaseUser.ts]: try to use status codes for every response

File: src/Week3/key.json
Comment (aditya-s-19) [inline src/Week3/key.json:1]: dont upload this file as its secret

**Review by anusreesubash — APPROVED** [review]

**Review by shreyansh-khatri — APPROVED** [review]
